src/core/keybindings.c

```diff
@@ -3176,9 +3206,6 @@ handle_toggle_tiled (MetaDisplay *display,
               next_cycle = META_TILE_CYCLE_25;
               break;
             case META_TILE_CYCLE_25:
-              next_cycle = META_TILE_CYCLE_100;
-              break;
-            case META_TILE_CYCLE_100:
```

Comment:
Why remove this? We just added this recently

Reply:
I didn't remove it must of happened when I ran the git clone command and was too lazy to start over and copied the file. sorry about that.

Reply:
I'm new to contributing to other project mainly only used github for personal projects. So yeah I'm a noob at this sorry. Not a noob at coding in C or C++ but at this whole github thing I am.